doc/android/Makefile.android

```diff
@@ -35,17 +35,17 @@ prepare: checkout-oboe checkout-cerbero
 
 .PHONY: checkout-oboe
 checkout-oboe: $(OBOE)
-	cd $(OBOE) && git checkout 9bf3943
+	cd $(OBOE) && git checkout 395f3d6ac25c2b069d53451b89dff4aa96d26eb8
 
 $(OBOE):
 	git clone https://github.com/Google/oboe.git $(OBOE) 
 
 .PHONY: checkout-cerbero
 checkout-cerbero: $(CERBERO)
-	cd $(CERBERO) && git checkout 0acd9b0
+	cd $(CERBERO) && git checkout e9f18b3b340de1648d885a0542d2f06c3fcfffe8
 
 $(CERBERO):
-	git clone https://github.com/atsushieno/cerbero.git $(CERBERO)
+	git clone https://github.com/falrm/cerbero.git $(CERBERO)
```

Comment:
Once the linked GStreamer GitLab MR is merged and propagated to GitLab, this can be changed to the official GSreamer cerbero as mentioned in the PR description.

Reply:
Ok, I'll keep an eye on that.